Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -245,21 +245,38 @@ void SetupTSOEmulation(FEXCore::Context::Context* CTX) {
 }
 } // namespace FEX::TSO
 
+/**
+ * @brief Get an FD from an environment variable and then unset the environment variable.
+ *
+ * @param Env The environment variable to extract the FD from.
+ *
+ * @return -1 if the variable didn't exist.
+ */
+static int GetFEXFDFromEnv(const char* Env) {
```

Comment:
Nit: could we rename this to `StealFEXFEDFromEnv`, because of the `unsetenv`? To me, "get" is a pure function whereas "steal" or "take" or "pop" destroys the input it returns.

Reply:
Renamed.